components/opentherm/schema.py

```diff
@@ -562,6 +566,7 @@ class SwitchSchema(EntitySchema):
                 "message": "STATUS",
                 "keep_updated": True,
                 "message_data": "flag8_hb_0",
+                "default_mode": "restore_default_on"
```

Comment:
I don't think it's a good idea to have CH and DHW on by default. I remember the first time I connected the interface board to my boiler, I made sure all switches are off just in case. Since these values are saved and restored from flash, it's really not much of a chore to turn them on manually once.

Reply:
Agreed. All switches, regardless of function, should default to OFF for safety reasons.
I had set default to ON in that commit to match what you had documented in the README.